source/vm/index.html.erb

```diff
@@ -36,153 +36,62 @@ title: All-In-One Virtual Machine for OpenShift Origin
     <div class="row wow info_vertical animated fadeInDown">
       <div class="col-md-12 text-left">
         <p>
-          Welcome to the all-in-one VM for
-          <a href="https://github.com/openshift/origin">OpenShift Origin</a>
-          , the FOSS PaaS from Red Hat. On this page you will find links to download:
+          Welcome to the instructions for running
+          <a href="https://github.com/openshift/origin" target="_blank">OpenShift Origin</a>
+          , the FOSS PaaS from Red Hat, in a local VM on your machine. On this page you will find instructions on how use <a href="https://github.com/minishift/minishift/blob/master/README.md" target="_blank">Minishift</a> for local application development with OpenShift.
```

Comment:
should read "how to use" rather than "how use". @wgordon17 please add another commit directly to thesteve0/master so that I can merge